cli.md

```diff
@@ -62,14 +62,14 @@ Also supports [GitHub Options](#github-options) and [Retry Options](#retry-optio
 * **-max-depth="int"** - How far to traverse the filesystem beneath the target
   directory for entrypoints. The default value is "-1".
 * **-pull-request-number="100"** - The GitHub pull request number associated with
-  this plan run. The default value is "0".
+  this plan. The default value is "0".
 * **-source-ref="ref-name"** - The source GitHub ref name for finding file changes.
 
-## Apply Run
+## Apply
 
 Run Terraform apply for a directory.
 
-Usage: guardian apply run [options] <directory>.
+Usage: guardian apply [options] <directory>.
```

Comment:
Can you update the `-dir` flag on all commands and remove the <directory> arg?

Reply:
done.